packages/cosmos_ui_components/lib/components/template/transaction_history_group_title.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:cosmos_ui_components/cosmos_text_theme.dart';
+import 'package:cosmos_ui_components/cosmos_theme.dart';
+import 'package:flutter/material.dart';
+
+class TransactionHistoryGroupTitle extends StatelessWidget {
+  final String title;
+
+  const TransactionHistoryGroupTitle({
+    Key? key,
+    required this.title,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: EdgeInsets.all(CosmosTheme.of(context).spacingL),
+      child: Text(title.split('-')[0], style: CosmosTextTheme.title0Bold),
```

Comment:
why do we need this title splitting here? what will happen if we pass a title that has no `-` inside?

Reply:
Yes, my bad. This split is because I am grouping the list by month+year with a '-' in between. This needs to go out to the consumer end. Will remove this from here.